index.html

```diff
@@ -250,6 +250,37 @@ <h3>LET'S CONNECT</h3>
                 </div>
             </div>
         </section>
+        <section id="contactcontainer">
+            <div class="rectangular1"></div>
+            <div class="rectangular2"></div>
+            <div class="circle2"></div>
+            <div class="circle1"></div>
+            <div class="ellipse2"></div>
```

Comment:
- [ ] [Optional] Images presented in your Figma template must be used as a background picture, not inside a dedicated HTML element.